src/backends/backend.js

```diff
@@ -26,7 +26,24 @@ class LocalStorageAuthStore {
 
 const slugFormatter = (template = "{{slug}}", entryData) => {
   const date = new Date();
-  const identifier = entryData.get("title", entryData.get("path"));
+
+  const getIdentifier = (entryData) => {
+	  const validIdentifierFields = ["title", "path"];
+	  const identifiers = validIdentifierFields.map((field) => {
+	  	return entryData.find((_, key) => {
+	  		return key.toLowerCase() === field;
```

Comment:
The braces and `return` keywords are unnecessary - this can be rewritten identically like so:

```js
const identifiers = validIdentifierFields.map((field) =>
  entryData.find((_, key) => key.toLowerCase() === field)
);
```